CMakeLists.txt

```diff
@@ -221,6 +221,11 @@ if(RDK_BUILD_PYTHON_WRAPPERS)
   if (PYTHONINTERP_FOUND AND NOT Python_ADDITIONAL_VERSIONS)
     set(Python_ADDITIONAL_VERSIONS "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
   endif (PYTHONINTERP_FOUND AND NOT Python_ADDITIONAL_VERSIONS)
+
```

Comment:
Perhaps give the last python2 release tag?

Reply:
It's very, very prominent in the README file